package.json

```diff
@@ -11,16 +11,19 @@
     "intl": "^1.2.5",
     "kolibri": "^0.15.4",
     "kolibri-design-system": "https://github.com/learningequality/kolibri-design-system#v1.3.1-beta0",
-    "vue": "^2.6.14",
-    "vue-intl": "3.0.0"
+    "vue": "2.7.14",
```

Comment:
We shouldn't upgrade the vue version here - should match Kolibri: https://github.com/learningequality/kolibri/blob/release-v0.16.x/kolibri/core/package.json#L34

Reply:
Totally, thanks. Reverted the version and tested it all again